vars/setupPMMClient.groovy

```diff
@@ -78,20 +78,20 @@ def call(String SERVER_IP, String CLIENT_VERSION, String PMM_VERSION, String ENA
                     cd ../
                     export PMM_CLIENT_BASEDIR=`ls -1td pmm2-client 2>/dev/null | grep -v ".tar" | head -n1`
                     export PATH="`pwd`/pmm2-client/bin:\$PATH"
-                    echo "export PATH=`pwd`/pmm2-client/bin:\$PATH" >> ~/.bash_profile
+                    echo "export PATH=`pwd`/pmm2-client/bin:\$PATH" >> ~/.bash_profile"
                     source ~/.bash_profile
                     pmm-admin --version
                     if [[ \$CLIENT_INSTANCE == yes ]]; then
                         if [[ \$ENABLE_PULL_MODE == yes ]]; then
-                            pmm-agent setup --config-file=`pwd`/pmm2-client/config/pmm-agent.yaml --server-address=\$SERVER_IP:443 --server-insecure-tls --server-username=admin --server-password=\$ADMIN_PASSWORD --metrics-mode=pull \$IP
+                          sudo pmm-agent setup --config-file=`pwd`/pmm2-client/config/pmm-agent.yaml --server-address=\$SERVER_IP:443 --server-insecure-tls --server-username=admin --server-password=\$ADMIN_PASSWORD --metrics-mode=pull \$IP
```

Comment:
The official [docs](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/index.html#register) do not say it need to be run with sudo, do they?